### PR TITLE
Support Python Django 3 & 4; updated readme.md

### PR DESCRIPTION
**Pull request by neilninja**
* Added support for Django version `3.2.13 LTS` and Django version `4.0.5`
* Tested on Python version `3.10.4`
* Updated `readme.md` to use Python `venv` and fixed instructions to find Moesif Application ID
* Removed 2 warnings
* Updated `.gitignore` for `db.sqlite3`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -215,3 +215,4 @@ fabric.properties
 # .idea/misc.xml
 # *.ipr
 .idea/
```
Comment (praves77) [inline .gitignore:217]: 1. You can delete all other .idea/<file-name> entries in line 168-181
2. Please also include `.DS_STORE` and `.vscode`

**Review by praves77 — APPROVED** [review]
Nice job. Looks fine to me.

File: mysite/polls/urls.py [hunk 2]
```diff
@@ -1,16 +1,16 @@
-from django.conf.urls import url
+from django.urls import re_path
```
Comment (keyur9) [inline mysite/polls/urls.py:1]: Could we test that the imports works with the previous version of Django as well? Like 2.x, and 1.9?

**Review by keyur9 — CHANGES_REQUESTED** [review]
few comments but lgtm otherwise

File: .gitignore (hunk 1 above)
Comment (neilninja) [reply .gitignore:202]: Thanks! Updated

File: mysite/polls/urls.py (hunk 2 above)
Comment (neilninja) [reply mysite/polls/urls.py]: Thanks you! I took your feedback and added support for older versions as well. Now it works for Django 1.11 LTS, 2.2 LTS, 3.2 TLS, and 4.0.5

File: mysite/mysite/settings.py [hunk 3]
```diff
@@ -188,3 +188,5 @@ def get_metadata(req, res):
 # https://docs.djangoproject.com/en/1.10/howto/static-files/
 
 STATIC_URL = '/static/'
+
+DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
```
[the remark this answers is not in the document]
Comment (neilninja) [reply mysite/mysite/settings.py:195]: Thanks! Added comment

**Review by keyur9 — APPROVED** [review]
lgtm

**Pull request merged**